### ISSUE TITLE
Error happened when use global eslint

### ISSUE THREAD

**Issue by Cherinho** [mradionov/eslint-plugin-disable#12]
I use globally installed eslint and eslint-plugin-angular. My project in disk D and npm in disk C. So when I trying to use eslint-plugin-disabled I got the error below

> Error on attempt to find parent and base ESLint configuration files of '.eslintrc.json': Cannot find module 'eslint' from 'D:\***\***'
			Error: Cannot find module 'eslint' from 'D:\***\***'
			at Function.module.exports [as sync] (C:\Users\***\AppData\Roaming\npm\node_modules\eslint-plugin-disable\node_modules\resolve\lib\sync.js:40:15)
			at Object.<anonymous> (C:\Users\***\AppData\Roaming\npm\node_modules\eslint-plugin-disable\index.js:26:26)
			at Module._compile (module.js:643:30)
			at Object.Module._extensions..js (module.js:654:10)
			at Module.load (module.js:556:32)
			at tryModuleLoad (module.js:499:12)
			at Function.Module._load (module.js:491:3)
			at Module.require (module.js:587:17)
			at require (internal/module.js:11:18)
			at Plugins.load (C:\Users\***\AppData\Roaming\npm\node_modules\eslint\lib\config\plugins.js:101:26)

Config file .eslintrc.json below
```
{
  "extends":[
    "plugin:angular/johnpapa"
  ],
  "plugins": [
    "disable"
  ]
}
```

**Comment by mradionov** [comment]
Thanks, I'll take a look.

**Comment by mradionov** [comment]
New release with the fix [1.0.1](https://github.com/mradionov/eslint-plugin-disable/releases/tag/v1.0.1) should now be available for download on [npm](https://www.npmjs.com/package/eslint-plugin-disable).

**Comment by Cherinho** [comment]
thanks a lot for quick fix
